Fixes/color palette

Gayathri-Anamala opened the pull request:
# TITLE = Updated colors used in the app (from Mui Color Palette)

## Description
Summary of changes

## PR Type
- [x] Bug Fix
- [ ] Feature
- [ ] Build Fix
- [ ] Testing
- [ ] General Improvement
- [ ] Cherry Pick

## Links
Jira: [Jira issue #](fill-in.org)

## Test Plan

## Screenshots

mrahanjam committed:
update workflow to run cucumber tests

radhikav1 committed:
Convert computeprofile tests cy to selenium.

mrahanjam committed:
Merge pull request #580 from cdapio/fix/CDAP-17935

fix partial comment loss

mrahanjam committed:
Merge pull request #576 from cdapio/build/update-workflow-to-run-cucumber-tests

Build/update workflow to run cucumber tests

radhikav1 committed:
Merge pull request #581 from cdapio/testing/CDAP-19221

[CDAP-19220]Conversion of compute profile tests from cypress to selenium

mrahanjam committed:
fix sandbox build issue

mrahanjam committed:
Merge pull request #584 from cdapio/fix/sandbox-build-issue

fix sandbox build issue

radhikav1 committed:
Convert controlcenter cy tests to selenium.

mrahanjam committed:
fix deploy maven issue

de-lan committed:
CDAP-19492: update composite trigger UI

mrahanjam committed:
Merge pull request #586 from cdapio/fix/deploy-maven-issue

fix deploy maven issue

radhikav1 committed:
Merge pull request #585 from cdapio/testing/CDAP-19221-controlcenter-tests

[CDAP-19221] Conversion of control center tests from cypress to selenium

mrahanjam committed:
update sandbox url in cloudbuild

mrahanjam committed:
Merge pull request #588 from cdapio/build/update-sandbox-url-in-cloudbuild

update sandbox url in cloudbuild

mrahanjam committed:
fix sandbox version parsing

seanfeld committed:
update node

seanfeld committed:
updated package json to reflect new node version

seanfeld committed:
don't need to specify engine anymore

mrahanjam committed:
Merge pull request #591 from cdapio/fix/sandbox-version-parsing

fix sandbox version parsing

mrahanjam committed:
update node version in workflow

seanfeld committed:
Merge pull request #592 from cdapio/pom/node-version

Pom/node version

sumengwang committed:
navbar test rebase

sumengwang committed:
Merge pull request #575 from cdapio/angular/react-migration

Merge Angular/react migration to Develop

sumengwang committed:
fixed existing cucumber tests

fix

sumengwang committed:
Merge pull request #596 from cdapio/bugfix/fix-react-migartion-cucumber-tests

fixed existing cucumber tests

sumengwang committed:
Merge pull request #595 from cdapio/testing/CDAP-19227

 Convert navbar.spec.ts test

sumengwang committed:
fix all but page.level.errors.spec.ts

remove hardcoded value

remove debugger

nits

sumengwang committed:
Merge pull request #597 from cdapio/fix/angular-to-react-cypress-tests-failure

Fix/angular to react cypress tests failure

sumengwang committed:
fix metadata url

remove outdated test

sumengwang committed:
Merge pull request #600 from cdapio/fix/metadata_url

fix metadata url

njbriggs committed:
CDAP-19503 Remove unused Angular network client code

seanfeld committed:
add functionality to add external links to the cog menu in the header bar

sumengwang committed:
add locators

fix height

use calc

sumengwang committed:
Merge pull request #605 from cdapio/fix/plugin-1350

add locators and change left panel height

sumengwang committed:
WIP edit page

a dumb way to do things..

allow deploy on edit from ui

add changeSummary box

put isEdit in params

save changeSummary to store

add parentversion

parentVersion

remove debugger

suspend schedule on deployment

prevent deploying unchanged pipeline

remove unnecessary schedule steps

fix some bugs

add error reducer

add reconciliation flow

add draft out of date status

style

poll edit status

some nits

hide edit button behind feature flag

higher level featureprovider

smritiDivami committed:
null missing

richadivami committed:
Missing and Null Value

richadivami committed:
Missing and Null Value Invalid

richadivami committed:
Missing and Null Value Invalid

richadivami committed:
Missing and Null Value Invalid empty cell

richadivami committed:
Missing and Null Value Invalid empty cell

seanfeld committed:
Merge pull request #604 from cdapio/toolbar/external-links

Add external links to toolbar cog menu

richadivami committed:
Giving variables a meaningful name GridTable

richadivami committed:
Merge branch 'feature/wrangle-2' of https://github.com/adityadivami/cdap-ui-forked into feature/wrangle-2

richadivami committed:
No record screen

richadivami committed:
No Record Arrow Function Format change

richadivami committed:
Moving to Utils

richadivami committed:
Moving to Utils Files

sumengwang committed:
fix

sumengwang committed:
Merge pull request #601 from cdapio/fix/flaky-joiner-test

fix flaky joiner test

adityadivami committed:
merged feature/wrangle

sumengwang committed:
fix joiner test

this should fix it

richadivami committed:
Format change

richadivami committed:
Merge branch 'feature/no-record-screen' of https://github.com/adityadivami/cdap-ui-forked into feature/no-record-screen

richadivami committed:
Adding Types to types.ts

richadivami committed:
Replacing for loop with map

richadivami committed:
Removing Most Frequent Item code from no-record-screen

richadivami committed:
Adding types styles and utils

richadivami committed:
Adding types styles and utils

richadivami committed:
Adding Default missing and null value

richadivami committed:
Adding Default missing and null value add

adityadivami committed:
Merge branch 'feature/wrangle' of github.com:adityadivami/cdap-ui-forked into feature/null-missing

Commit:
feat: No sample data found screen

Commit:
Merge branch 'feature/wrangle' of https://github.com/adityadivami/cdap-ui-forked into feature/no-record-screen

Commit:
fix: removed scroll on no records screen

adityadivami committed:
fix: removed mock metricsJSON

radhikav1 committed:
wrangler spanner cy to selenium tests

sumengwang committed:
Merge pull request #608 from cdapio/fix/flaky-joiner-test

Fix flaky joiner test

mrahanjam committed:
make env vars available to e2e tests in workflow

mrahanjam committed:
Merge pull request #612 from cdapio/fix/make-env-vars-available-to-e2e-tests

make env vars available to e2e tests in workflow

Commit:
fix: routing modifications

Commit:
fix: addressed PR comments

de-lan committed:
Merge pull request #587 from cdapio/feature/CDAP-19492/composite_trigger_upgrade

CDAP-19492: update composite trigger UI

sumengwang committed:
remove style

de-lan committed:
CDAP-19230: convert pipeline canvas actions tests

sumengwang committed:
Merge pull request #614 from cdapio/bugfix/get-schema-style

fix get schema style

sumengwang committed:
switch back tab after deleting trigger

njbriggs committed:
Update README, CONTRIBUTING, and DEVELOPERS files to match current practices

njbriggs committed:
Merge pull request #616 from cdapio/task-ui/cdap-19503

CDAP-19503 Remove unused Angular network client code

njbriggs committed:
Merge pull request #617 from cdapio/update-contributing-file

Update README, CONTRIBUTING, and DEVELOPERS files to match current practices

radhikav1 committed:
Merge pull request #610 from cdapio/testing/CDAP-19259

wrangler spanner cy to selenium tests

richadivami committed:
Fixes

sumengwang committed:
Merge pull request #615 from cdapio/bugfix/trigger-flaky-test

fix flaky trigger test

Commit:
ConnectionListColorChanges

de-lan committed:
Merge pull request #619 from cdapio/test/CDAP-19230

CDAP-19230: convert pipeline canvas actions tests

sumengwang committed:
convert nuxtour test

add whitespace

fix npe

format

de-lan committed:
chore(CDAP-19229): convert page.level.errors.spec.ts test from cypress to cucumber selenium

sumengwang committed:
concert logviewer test

checkstyle

add run pipeline

try this

try this

create tmp dir first

clean up

add screenshots on failure

fix

checkstyle

fix

fix

change snapshot path

clean up

export cucumber reports

make test cleaner

try this

sumengwang committed:
concert logviewer test

checkstyle

add run pipeline

try this

try this

create tmp dir first

clean up

add screenshots on failure

fix

checkstyle

fix

fix

change snapshot path

clean up

export cucumber reports

make test cleaner

try this

adityadivami committed:
updated colors

de-lan committed:
Merge pull request #624 from cdapio/test/CDAP-19229

chore(CDAP-19229): convert page.level.errors.spec.ts test from cypress to cucumber selenium

sumengwang committed:
Merge pull request #623 from cdapio/testing/CDAP-19228

[CDAP-19228] convert nuxtour test

njbriggs commented on app/cdap/components/ConnectionList/styles.ts:
```diff
@@ -15,6 +15,8 @@
  */
 
 import { makeStyles } from '@material-ui/styles';
+import { blue } from '@material-ui/core/colors';
```
Nit: `import { blue, gray } from '@material-ui/core/colors';`

njbriggs replied:
And throughout

richadivami replied:
Okay, fixed

richadivami committed:
Adding JSDOc comment

richadivami committed:
Adding JS doc

richadivami committed:
Adding JS Doc comment

richadivami committed:
Pr fixes, typo correction

richadivami committed:
PR fixes blue, grey import

richadivami committed:
Merge branch 'fixes/color-palette' of https://github.com/adityadivami/cdap-ui-forked into fixes/color-palette

richadivami committed:
Type notations

richadivami committed:
Merging with feature/wrangle

richadivami committed:
Merge with feature/wrangle

sumengwang committed:
mvn clean won't remove node_modules

create new profile

change dev-clean clean config

njbriggs committed:
CDAP-19588 Switch toppanel-ctrl polling to prepare for Websocket removal

mrahanjam committed:
add data-testid tag for plugin properties

mrahanjam committed:
Merge pull request #630 from cdapio/chore/add-data-test-id-tag-for-plugin-properties

add data-testid tag for plugin properties with toggle switch

njbriggs committed:
Merge pull request #629 from cdapio/task-ui/cdap-19588

CDAP-19588 Switch toppanel-ctrl polling to prepare for Websocket removal

sivaRajanaDivami committed:
Missing/null values and frequent records

de-lan committed:
chore(CDAP-19231): convert complex schema tests to cucumber selenium tests

sivaRajanaDivami committed:
modified the export of noRecordScreen index file

sivaRajanaDivami committed:
fix: routing modifications

adityadivami committed:
udpated code to get workspace name

adityadivami committed:
removed console logs

sivaRajanaDivami committed:
Merge branch 'fixes/color-palette' of https://github.com/adityadivami/cdap-ui-forked into fixes/color-palette

sumengwang committed:
Merge pull request #628 from cdapio/improvement/e2e-tests-profile

mvn clean on dev-clean profile won't remove node_modules

adityadivami committed:
feat: added 1 testcase in GridTable

ManishaGundapuneedi committed:
Add custom tooltip tests

ManishaGundapuneedi committed:
Add tests for ongoing exploration card and remove unused code in wrangler home

adityadivami committed:
feat: added test cases to generateDataForExplorationCard util function

ManishaGundapuneedi committed:
Add tests for custom tooltip

smritiDivami committed:
Unit Test Cases

ManishaGundapuneedi committed:
Add tests for connection tabs

ManishaGundapuneedi committed:
Fix breadcrumb test issue

ManishaGundapuneedi committed:
Add tests for tab label components

smritiDivami committed:
Added Test Case

sivaRajanaDivami committed:
Test: connections-list page's UNIT TESTING

sivaRajanaDivami committed:
Test: connections-list page's UNIT TESTING

sivaRajanaDivami committed:
Test: connections-list page's UNIT TESTING

sivaRajanaDivami committed:
UNIT TESTING: grid table

sivaRajanaDivami committed:
unit test code got added

sivaRajanaDivami committed:
UNIT TESTING code got added

sivaRajanaDivami committed:
unit testing code got added

sivaRajanaDivami committed:
unit testing code got added

sivaRajanaDivami committed:
Merge branch 'fixes/grid-page-breadcrumb' of https://github.com/adityadivami/cdap-ui-forked into fixes/grid-page-breadcrumb

sivaRajanaDivami committed:
unit testing code got added

sivaRajanaDivami committed:
fixed breadcrumb

sivaRajanaDivami committed:
fixed a bug

adityadivami committed:
updated tests for connections list

adityadivami committed:
updated tests for connections list

adityadivami committed:
updated tests for connections list

adityadivami committed:
feat: updated on going data explorations test cases

adityadivami committed:
Merge branch 'test/connections-list' of github.com:adityadivami/cdap-ui-forked into test/connections-list

adityadivami committed:
Merge branch 'test/connections-list' of github.com:adityadivami/cdap-ui-forked into test/connections-list

adityadivami committed:
formatting changes

adityadivami committed:
formatting changes

adityadivami committed:
feat: updated test names in ConnectionTabs component

adityadivami committed:
feat: updated test names in ConnectionTabs component

adityadivami committed:
feat: updated test

adityadivami committed:
feat: updated test

adityadivami committed:
added copyrights to all files

adityadivami committed:
added copyrights to all files

adityadivami committed:
added copyrights to all files

adityadivami committed:
Merge branch 'test/connections-list' of github.com:adityadivami/cdap-ui-forked into test/connections-list

adityadivami committed:
feat: updated test

adityadivami committed:
feat: updated test

adityadivami committed:
feat: updated test

adityadivami committed:
feat: updated test

adityadivami committed:
feat: added copyrights to new files

adityadivami committed:
formatting changes

adityadivami committed:
formatting auto changes

adityadivami committed:
formatting auto changes

adityadivami committed:
Merge branch 'test/connections-list' of github.com:adityadivami/cdap-ui-forked into testing/grid-table

adityadivami committed:
formatting auto changes

adityadivami committed:
Merge branch 'testing/grid-table' of github.com:adityadivami/cdap-ui-forked into feature/no-record-screen

adityadivami committed:
formatting auto changes

adityadivami committed:
Merge branch 'feature/no-record-screen' of github.com:adityadivami/cdap-ui-forked into feature/null-missing

adityadivami committed:
Merge branch 'feature/no-record-screen' of github.com:adityadivami/cdap-ui-forked into feature/null-missing

adityadivami committed:
Merge branch 'feature/null-missing' of github.com:adityadivami/cdap-ui-forked into fixes/grid-page-breadcrumb

adityadivami committed:
Merge branch 'fixes/grid-page-breadcrumb' of github.com:adityadivami/cdap-ui-forked into fixes/color-palette

de-lan committed:
Merge pull request #632 from cdapio/test/CDAP-19231

chore(CDAP-19231): convert complex schema tests to cucumber selenium tests

seanfeld commented on app/cdap/components/ConnectionList/Components/TabLabelCanSample/__test__/TabLabelCanSimple.test.tsx:
```diff
@@ -0,0 +1,77 @@
+  category: 'File',
+  description: 'Connection to browse and sample data from the local file system.',
+  className: 'io.cdap.plugin.batch.connector.FileConnector',
+  // canBrowse: false,
```
why are these commented out?

seanfeld commented on app/cdap/components/ConnectionList/Components/TabLabelCanSample/__test__/TabLabelCanSimple.test.tsx:
```diff
@@ -0,0 +1,77 @@
+  expect(setIsErrorOnNoWorkSpace).toHaveBeenCalled();
+});
+
+// test('Fire click event and trigger onCreateWorkspace function ', () => {
```
? please leave a reason why they're commented out if they're left in

seanfeld commented on app/cdap/components/ConnectionList/Components/TabLabelCanSample/index.tsx:
```diff
@@ -106,27 +106,33 @@ export default function TabLabelCanSample({
-              {WRANGLE_LABEL}
-            </Typography>
-          </Box>
+        <button
```
can you use the material ui button for this

seanfeld commented on app/cdap/components/ConnectionList/Components/TabLabelCanSample/index.tsx:
```diff
@@ -106,27 +106,33 @@ export default function TabLabelCanSample({
         {label}
       </Typography>
-      <button className="wranglingHover" onClick={() => onExplore(entity)}>
+      <button
```
material ui button or one of the button components that we have for this?

sumengwang committed:
Merge pull request #606 from cdapio/feature/CDAP-19524

[CDAP-19524] LCM edit page

sumengwang committed:
switch location due to style change

sumengwang committed:
Merge pull request #633 from cdapio/fix/pipeline-metadata-tag-location

Fix name and description unclickable during test

sumengwang committed:
add edit stage column

turn featureflag on

basic editing stage

add discard modal to pipeline details page

add feature flag

refactor excessive api calls

remove unused imports

address comments

minor things

move edit status to draft list

add ophraned draft deploy logic

rename component

sumengwang committed:
Merge pull request #625 from cdapio/testing/CDAP-19226

[CDAP-19226] convert logviewer test

njbriggs committed:
CDAP-19009 Switch Studio to use HTTP instead of Websockets

njbriggs committed:
CDAP-19009 Switch Studio to use HTTP instead of Websockets

mrahanjam committed:
fix CDAP-19625

mrahanjam committed:
fix CDAP-19625

mrahanjam committed:
Merge pull request #636 from cdapio/fix/CDAP-19625

fix CDAP-19625

adityadivami committed:
feat: moved strings to constants

adityadivami committed:
feat: removed unwanted interfaces

adityadivami committed:
feat: removed unwanted logic

adityadivami committed:
feat: default exports for components

adityadivami committed:
Merge branch 'feature/null-missing' of github.com:adityadivami/cdap-ui-forked into feature/null-missing

njbriggs committed:
CDAP-19559 Add deprecated messaging to Checkpointing for streaming pipelines

njbriggs committed:
CDAP-19559 Add deprecated messaging to Checkpointing for streaming pipelines

sumengwang committed:
fix cypress tests

only check error exists

change it back

sumengwang committed:
fix cypress tests

only check error exists

change it back

sumengwang committed:
Merge pull request #639 from cdapio/test-fix/reference-name-not-required

Fix expected error count

sumengwang committed:
forgot to remove referenceName error check

sumengwang committed:
forgot to remove referenceName error check

sumengwang committed:
Merge pull request #640 from cdapio/test-fix/reference-name-not-required

fix test - forgot to remove referenceName error check

sivaRajanaDivami committed:
Reviewed

sivaRajanaDivami committed:
code reviewed

adityadivami committed:
Merge branch 'feature/null-missing' of github.com:adityadivami/cdap-ui-forked into fixes/grid-page-breadcrumb

adityadivami committed:
Merge branch 'fixes/grid-page-breadcrumb' of github.com:adityadivami/cdap-ui-forked into fixes/color-palette

njbriggs committed:
Fix error code handling in HTTP Pipeline code

njbriggs committed:
Fix error code handling in HTTP Pipeline code

njbriggs committed:
Switch versions and namespaces requests to HTTP

njbriggs committed:
Switch versions and namespaces requests to HTTP

njbriggs committed:
Merge pull request #638 from cdapio/task-ui/cdap-19559

CDAP-19559 Add deprecated messaging to Checkpointing for streaming pipelines

njbriggs committed:
Merge pull request #634 from cdapio/feature-ui/cdap-19009-2

CDAP-19009 Switch Studio to use HTTP instead of Websockets

sumengwang committed:
fix status code

sumengwang committed:
fix status code

sumengwang committed:
Merge pull request #641 from cdapio/test-fix/undefined-statuscode

fix undefined status code

njbriggs committed:
Add more detail to DEVELOPERS on I18n and testing

njbriggs committed:
Add more detail to DEVELOPERS on I18n and testing

njbriggs committed:
Merge pull request #646 from cdapio/update-contributing-file-2

Add more detail to DEVELOPERS on I18n and testing

mrahanjam committed:
CDAP-19637: display rejected requests on hdf side

de-lan committed:
fix(CDAP-19621): Save button covering the profile list

de-lan committed:
fix(CDAP-19621): Save button covering the profile list

de-lan committed:
Merge pull request #648 from cdapio/bugfix/CDAP-19621

fix(CDAP-19621): Save button covering the profile list

seanfeld committed:
creates templates and deletes templates and change versions for templates

adityadivami committed:
Merged feature/wrangle

mrahanjam committed:
Merge pull request #647 from cdapio/feat/CDAP-19637

CDAP-19637: display rejected requests on hdf side

adityadivami committed:
Merge branch 'feature/wrangle-first-pr' of github.com:adityadivami/cdap-ui-forked into test/wrangler-home

adityadivami committed:
Merge branch 'test/wrangler-home' of github.com:adityadivami/cdap-ui-forked into test/connections-list

adityadivami committed:
Merge branch 'test/connections-list' of github.com:adityadivami/cdap-ui-forked into testing/grid-table

adityadivami committed:
Merge branch 'testing/grid-table' of github.com:adityadivami/cdap-ui-forked into feature/no-record-screen

adityadivami committed:
Merge branch 'feature/no-record-screen' of github.com:adityadivami/cdap-ui-forked into feature/null-missing

adityadivami committed:
Merge branch 'feature/null-missing' of github.com:adityadivami/cdap-ui-forked into fixes/grid-page-breadcrumb

adityadivami committed:
Merge branch 'fixes/grid-page-breadcrumb' of github.com:adityadivami/cdap-ui-forked into fixes/color-palette

seanfeld committed:
creates templates and deletes templates and change versions for templates

small fixes

forgot headers

comments

open studio page differently

add wait time for rendering

seanfeld committed:
Merge branch 'develop' of github.com:cdapio/cdap-ui into podesta/studio-plugin-verisons-create-template

sumengwang committed:
filter config groups before assign filtered values

different approach to handle toggle value change

fix

sumengwang committed:
Merge pull request #637 from cdapio/feature/CDAP-19479

[CDAP-19479] Add editing stage column to deployed pipeline list

seanfeld committed:
change rich tooltip to index

seanfeld committed:
Merge pull request #649 from cdapio/podesta/studio-plugin-verisons-create-template

Create template/change version/delete template for studio plugin [REACT]

njbriggs committed:
CDAP-19661 Remove bcookie from browsers

njbriggs committed:
Merge pull request #655 from cdapio/bugfix-ui/cdap-19661-2

CDAP-19661 Remove bcookie from browsers - retry CLA

adityadivami committed:
formatting auto fixes

adityadivami committed:
formatting auto fixes

adityadivami committed:
Merge branch 'feature/no-record-screen' of github.com:adityadivami/cdap-ui-forked into feature/null-missing

adityadivami committed:
merge

adityadivami committed:
Merge branch 'fixes/grid-page-breadcrumb' of github.com:adityadivami/cdap-ui-forked into fixes/color-palette

sumengwang committed:
Merge pull request #652 from cdapio/bugfix-ui/CDAP-19653

[CDAP-19653] filter config groups before assign filtered values

adityadivami committed:
Merge pull request #233 from cdapio/develop

Pull latest from parent repo to divami forked repo

adityadivami committed:
Merge branch 'develop' of github.com:adityadivami/cdap-ui-forked into feature/wrangle-first-pr

adityadivami committed:
Merge branch 'feature/wrangle-first-pr' of github.com:adityadivami/cdap-ui-forked into test/wrangler-home

adityadivami committed:
Merge branch 'test/wrangler-home' of github.com:adityadivami/cdap-ui-forked into test/connections-list

adityadivami committed:
Merge branch 'test/connections-list' of github.com:adityadivami/cdap-ui-forked into testing/grid-table

adityadivami committed:
Merge branch 'testing/grid-table' of github.com:adityadivami/cdap-ui-forked into feature/no-record-screen

adityadivami committed:
Merge branch 'feature/no-record-screen' of github.com:adityadivami/cdap-ui-forked into feature/null-missing

adityadivami committed:
Merge branch 'feature/null-missing' of github.com:adityadivami/cdap-ui-forked into fixes/grid-page-breadcrumb

sudhadivami committed:
Merge branch 'fixes/grid-page-breadcrumb' of https://github.com/adityadivami/cdap-ui-forked into fixes/color-palette

dlakshmi123 committed:
resolved_conflicts

gupteswar-7008 committed:
formatting auto fixes

abhilashDivami committed:
Merge branch 'feature/no-record-screen' of https://github.com/adityadivami/cdap-ui-forked into feature/null-missing

abhilashDivami committed:
Merge branch 'feature/null-missing' of https://github.com/adityadivami/cdap-ui-forked into fixes/grid-page-breadcrumb

abhilashDivami committed:
Merge branch 'fixes/grid-page-breadcrumb' of https://github.com/adityadivami/cdap-ui-forked into fixes/color-palette
